engineering/methodology.md

```diff
@@ -1,131 +1,196 @@
+there is no separate column and Tag is used instead, to signify if something is
+blocked.
 
-```
-SM = Scrum Master
-PO = Product Owner
-```
+More details in [Definition](#definition) section below.
 
-Team          | Roles                   | Sprint length | Sprint Planing
---------------|-------------------------|---------------|---------------
-Platform      | SM:Sergio, PO: Marcelo  | 1w            | Monday, 09:30
-Data Retrieval| SM:Santiago, PO: Maximo | 2w            | Tuesday,10:00
-ML            | SM:Vadim, PO: Maximo    | 2w            | Monday, 11:30
-Infra         | SM:Maximo, PO: Maximo   | 2w            | Monday, 15:00
+### How to pick next issues to worked on?
 
-*General Standup*: 10:30 @ Margaret Hamilton room
+Each team decides what works best for them. One effective way is picking tasks
+from **right to left** in the Kanban board, choosing from those issues that are
+neither assigned to someone nor blocked. A single person should not be in charge
+of delivering the whole issue from Development to Done. In that way knowledge is
+better spread across the whole team.
 
+For this to work properly, issues must be documented so that a person can move on with the issue, reading what has been done.
 
-### Standups
-Every day, except for OSF and Planing.
 
-Team          | Time      |  Room
---------------|-----------|-------------
-Platform      | 10:00     | Ada Lovelace
-Data Retrieval| 10:00     | Margaret Hamilton
-ML            | 10:15     | open space
-Infra         | ???       | Margaret Hamilton
+## Leads Meeting
 
+Same as something called SoS before - a weekly meeting on Tuesdays 14:30 of CTO
+\w all team leads to catch up on ongoing progress.
 
-### Planning
+Each team lead will prepare:
 
-Planning meetings
+* Review on what has been worked last week
+* What are the things we are looking forward to? (re-prioritize things)
+* Pitch new ideas from our workflow
 
-  + It is really a planing DAY, not just a meeting:
+Then grooming company’s backlog/epics from <> is conducted.
 
-    * There will be a 1 hour long kickoff meeting at 10:00 (first team) and
-      11:00 (second team)
+## Team Meeting
 
-    * The rest of the day, the team will think about proposed sprint plan and
-      decide if it is doable.
+A Bi-weekly meeting of the CTO with the team lead and optionally with the full
+team (to be decided by the lead), in order to comment the status of the
+projects. Scheduled with each team individually by CTO.
 
-         - leads review backlog before the planning
-         - read backlog
-         - agree on scope of sprint
-         - estimate
+## Standup
 
-    * There will be a close meeting at 17:30 (first) and 18:00 (second) to confirm
-      the sprint plan.
+All standups are happening online and asynchronously, per-team in appropriate
+`su-*` Slack channels.
 
-  + All OKR objectives for each team member will be covered in the planning.
+## Demo
 
+Demo meetings are monthly event.
```

Comment:
"are a monthly event" or "are monthly events".

Reply:
Or "Demo meeting is a monthly event"